core/src/test/java/apoc/export/cypher/ExportCypherMultiRelTest.java

```diff
@@ -93,74 +91,63 @@ public void teardown() {
 
     @Test
     public void updateAllOptimizationNone() {
-        String expectedCypherStatement =
-                NODES_MULTI_RELS + SCHEMA_WITH_UNIQUE_IMPORT_ID + RELS_MULTI_RELS + CLEANUP_SMALL_BATCH;
+        String expectedCypherStatement = NODES_MULTI_RELS + RELS_MULTI_RELS + CLEANUP_SMALL_BATCH;
         final Map<String, Object> map = withoutOptimization(map("cypherFormat", "updateAll"));
 
         testsCommon(expectedCypherStatement, map);
     }
```

Comment:
These tests were weird. They used to have schema modification queries in them that are silently skipped in `runMany`.